### ISSUE TITLE
Creating README?

### ISSUE THREAD

**Issue by AmirNaghibi** [lighthouse-labs/todo-list-js-exercise#4]
I was looking for a simple to-do list app in js. This seems to be somewhat related to what I am looking for. 

Is this a starter project? 

Do you need any help to improve this repo? such as creating documentation or making it more interactive?

**Comment by oggy107** [comment]
yeah you can try to add pull request after adding a README.md file or add a simple ui with html and css

**Comment by danielolasupo02** [comment]
I love your project and I'd love to refactor the code to look much cleaner.

**Comment by tusharverma200** [comment]
# Do you want us to create a readme for your project?

**Comment by AmirNaghibi** [comment]
Thanks for the interest! feel free to create a README and create a PR and tag the repo owner.
